protocol/avdl/keybase1/simple_fs.avdl

```diff
@@ -307,4 +307,17 @@ protocol SimpleFS {
    recorded by the server) of their most recent edit.
    */
   FSFolderEditHistory simpleFSFolderEditHistory(Path path);
+
+  // suppressNotifications suppresses KBFS notifications, particularly FSEvent
+  // and FSSyncEvent.
+  //
+  // Caller should call this again after nextSuppressIn to continue
+  // suppressing.  If no such call is made in 2*nextSuppressIn, KBFS will
+  // resume notifications.
+
+  // When resuming notifications, if a notification would have happened since
+  // last suppressNotifications call but was suppressed, this notification will
+  // go through. If multiple notifications would have happened, only one of
+  // them is sent at resuming.
```

Comment:
Why does it make sense to send one event?  Seems kind of arbitrary, since all the events are independent.

Reply:
At this moment we treat all notifications equally without inspecting its content. We just use it as a signal to start the polling, so it wouldn’t break anything if we miss a couple of them. Sending all the missed one one the other hand is a bit wasteful. Perhaps we should change the notification semantics or add a new (that doesn’t make it distinct from each other) one to avoid confusion?

Reply:
I don't want to change semantics in this PR.  I guess just explain all this in the comment so it makes it clear that the listener shouldn't depend on any differential nature of the data between different notifications.